packages/libs/react-ui/src/components/Typography/Heading/Heading.tsx

```diff
@@ -1,78 +1,72 @@
+import classNames from 'classnames';
 import type { ComponentPropsWithRef, FC } from 'react';
 import React from 'react';
-import type {
-  colorVariants,
-  fontVariants,
-  transformVariants,
-} from '../typography.css';
-import type { elementVariants } from './Heading.css';
-import { heading } from './Heading.css';
+import {
+  fontH1Bold,
+  fontH1Regular,
+  fontH2Bold,
+  fontH2Regular,
+  fontH3Bold,
+  fontH3Regular,
+  fontH4Bold,
+  fontH4Regular,
+  fontH5Bold,
+  fontH5Regular,
+  fontH6Bold,
+  fontH6Regular,
+} from '../../../styles';
```

Comment:
you can also write this like the following
```suggestion
} from '@styles/tokens/styles';
```

Reply:
I know, but this one is clearer and I also don't like the false negative linting error in my editor also it easier to use the relative path to make it clear where is the import is coming from and also make go to the declaration editor functionality work as expected